app/Http/Requests/StoreGroupRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+use App\Models\Group;
+use Auth;
+
+class StoreGroupRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
I assume authorized probably should do something here, like check to see if the user is logged in maybe?  I'm not altogether sure, since that's already handled by the middleware though.